src/SwedbankPay.Sdk/Payments/CardPayments/ICardPaymentsResource.cs

```diff
@@ -13,6 +13,13 @@ public interface ICardPaymentsResource
         /// <returns></returns>
         Task<CardPayment> Create(CardPaymentRequest paymentRequest, PaymentExpand paymentExpand = PaymentExpand.None);
 
+        /// <summary>
+        ///     Initiates a new credit card verify
+        /// </summary>
+        /// <param name="paymentVerifyRequest"></param>
+        /// <param name="paymentExpand"></param>
+        /// <returns></returns>
+        Task<CardVerify> Verify(CardPaymentVerifyRequest paymentVerifyRequest, PaymentExpand paymentExpand = PaymentExpand.None);
```

Comment:
Codacy found an issue: [Use the overloading mechanism instead of the optional parameters.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-dotnet/pullRequest?prid=5742807)